packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -78,15 +78,17 @@ const AuthenticatedApp = () => {
 
   const shouldShowLoader = isLoading || shouldShowNotDependentQueriesLoader;
 
+  const adminUserId = hashAdminUserEmail(userInfo);
   const appInfosValue = useMemo(() => {
```

Comment:
Can we remove the use of memo? I don't personally see the point in using it to construct an object with no transformations?

Reply:
Is there any harm in keeping it in your opinion? I don't personally know why it is used here, but this lack of context could delay this PR even further in case it needs to be reverted back, that's my only concern.

Reply:
This optimisation isn't free and in this circumstance I believe it's probably doing more harm than good. In the FE sync and off the back it was agreed we would remove these premature optimisations.

> I don't personally know why it is used here

No one does, that's the other problem.

Reply:
Gotcha, thanks. I wasn't at the FE sync, so didn't know that was agreed upon. In that case I can refactor it and give it a try

Reply:
Thanks @ivanThePleasant 😄

Reply:
Hey @joshuaellis annoyingly we have some eslint rules that will scream at me if I want to get rid of this memo, and that will need to be fixed via our eslint package I guess. Because this is not part of the scope of this PR, I'll leave it as is for now, but will remove it in a separate PR